lib/packages_groups_xml_creator.py

```diff
@@ -18,6 +18,20 @@
 
 COMPS_DOCTYPE = "<!DOCTYPE comps PUBLIC \"-//HostOS//DTD Comps info//EN\" \"comps.dtd\">"
 
+def convert_name_to_id(element_name, suffix):
```

Comment:
minor side comment: appending a suffix is out of the scope of "converting a name to ID", but as this feature is reused, OK